dokka-subprojects/core/src/main/kotlin/org/jetbrains/dokka/links/DRI.kt

```diff
@@ -146,6 +146,10 @@ public data class Nullable(val wrapped: TypeReference) : TypeReference() {
     override fun toString(): String = "$wrapped?"
 }
 
+public data class Vararg(val elementType: TypeReference) : TypeReference() {
+    override fun toString(): String = "Vararg[$elementType]"
```

Comment:
Unfortunately, this is used in `package-list` somehow

https://github.com/Kotlin/dokka/blob/d4d457d357627fd8100e4f0d47868020485996ec/dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/resolvers/external/DefaultExternalLocationProvider.kt#L20C53-L20C67 


This may be left as it was previously, but I'm not sure.

Reply:
If you are talking about `package-list`s that are currently in the integration tests, they seem to work